bin/npx

```diff
@@ -32,4 +36,9 @@ case `uname` in
     ;;
 esac
 
+if [ isexe==1 ] && [ -f "/bin/wslpath" ]; then # run the corresponding cmd instead
```

Comment:
It'd be better to do `if [ $isexe -eq 1 ] && type -p wslpath 2>/dev/null ; then`.

`-eq` forces it to be a numeric value.  `type -p` will determine that `wslpath` is a file executable in the `$PATH`.  (And it's a bash builltin that can be faster than `which` in some scenarios.)